neon_tts_plugin_mozilla_remote/__init__.py

```diff
@@ -19,54 +19,32 @@
 # WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 # USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from urllib.parse import urlencode
-from urllib.request import urlopen
-from neon_utils.configuration_utils import get_neon_tts_config
-from neon_utils.logger import LOG
-from neon_utils.parse_utils import format_speak_tags, normalize_string_to_speak
+import requests
 
-try:
-    from neon_audio.tts import TTS, TTSValidator
-except ImportError:
-    from mycroft.tts import TTS, TTSValidator
-from mycroft.metrics import Stopwatch
+from ovos_plugin_manager.templates.tts import TTS, TTSValidator
 
 
 class MozillaRemoteTTS(TTS):
 
     def __init__(self, lang="en-us", config=None):
-        config = config or get_neon_tts_config().get("mozilla", {})
-        super(MozillaRemoteTTS, self).__init__(lang, config, MozillaTTSValidator(self),
-                                               audio_ext="wav",
-                                               ssml_tags=["speak"])
-        self.base_url = config.get("api_url", "http://0.0.0.0:5002/api/tts")
-
-    def get_tts(self, sentence, wav_file, speaker=None):
-        stopwatch = Stopwatch()
-
-        to_speak = format_speak_tags(sentence, False)
-        LOG.debug(to_speak)
-        if to_speak:
-            url = self._build_url(normalize_string_to_speak(to_speak))
-            with stopwatch:
-                wav_data = urlopen(url).read()
-            LOG.debug(f"Request time={stopwatch.time}")
-
-            with stopwatch:
-                with open(wav_file, "wb") as f:
-                    f.write(wav_data)
-            LOG.debug(f"File access time={stopwatch.time}")
+        super().__init__(lang, config,
+                         RemoteMozillaTTSValidator(self),
+                         audio_ext="mp3")
+        self.url = self.config.get("api_url") or \
+                   self.config.get("url", "http://0.0.0.0:5002/api/tts")  # mycroft-core compat
+
+    def get_tts(self, sentence, wav_file):
```

Comment:
Should this accept args/kwargs for backwards compat?

Reply:
no need, only those 2 args are part of the official api, other args are optional

execute and synth parse kwarsg and signature internally and only pass the ones accepted by this method, direct usage can be a concern however.... but thats breaking the api? bump a version is enough?

Reply:
Ah, right. I forgot we are patching this differently. Yes, a major version bump will address it